### ISSUE TITLE
Discord bot cannot get novel url in channel chat

### ISSUE THREAD

**Issue by yudilee** [dipu-bd/lightnovel-crawler#65]
Discord bot works great in 1 on 1 chat but not on channel chat because it can read novel url or search item in chat. Unlike telegram bot, in discord while in channel they don't have reply feature. Maybe we need shorter command feature for chat in channel but also retain conversation like request for 1 on 1 chat. shorter comman is similar to argument in console bot for example 
for searching we can do :
!lncrawl search novel_title novel_source --> to generate novel url
for generating book
!lncrawl format_book novel_url pack_by_volume all -> to generate all chapter in format_book format
etc

**Comment by dipu-bd** [comment]
As channel chat is public, handling different users can be troublesome. I am not sure if there is a way to display a private message only to a specific user in a channel chat. I had to use 1 on 1 chat so that one user won't get confused by seeing another ones message.

However, we could enable an one-liner initial command like you said for discord bot. I did not try to add complex commands because explaining the usage would be tiresome.
